pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/AccessorMethodGenerationRule.java

```diff
@@ -19,7 +19,6 @@
 import net.sourceforge.pmd.lang.java.symbols.JClassSymbol;
 import net.sourceforge.pmd.lang.java.symbols.JConstructorSymbol;
 import net.sourceforge.pmd.lang.java.symbols.JFieldSymbol;
-import net.sourceforge.pmd.lang.java.symbols.JMethodSymbol;
```

Comment:
less is more: no need for casting this way and onetime usages always feel to me like: `(RSPEC-S1488): Local variables should not be declared and then immediately returned or thrown`